data-pipeline/src/data_pipeline/data_types/seeds.py

```diff
@@ -0,0 +1,13 @@
+# This file consists of shared variables to be used by various pipeline
+#   functions if given the '--create-test-datasets' flag.
+
+SUBSAMPLE_FRACTION = 0.01
+INTEGER = 1337
```

Comment:
Nice. 

However, a bit confusing to name this `INTEGER` because someone could think this is a type and not a hardcoded value. Maybe something like `SEED_NUMBER`.

Reply:
of all the random things from my youth to survive to this point..."1337" and "leet".